pkg/alertmanager/api_test.go

```diff
@@ -62,13 +68,42 @@ receivers:
 		},
 	}
 
+	am := &MultitenantAlertmanager{
+		store:  noopAlertStore{},
+		logger: util.Logger,
+	}
 	for _, tc := range testCases {
-		cfgDesc := alerts.ToProto(tc.cfg.AlertmanagerConfig, tc.cfg.TemplateFiles, "fake")
-		err := validateUserConfig(cfgDesc)
+		payload, err := yaml.Marshal(&tc.cfg)
+		require.NoError(t, err)
+
+		req := httptest.NewRequest("GET", "http://foooo", bytes.NewReader(payload))
```

Comment:
```suggestion
		req := httptest.NewRequest(http.MethodPost, "http://alertmanager/api/v1/alerts", bytes.NewReader(payload))
```

Reply:
Apologies as I understand this is pretty minimal but I find it helps illustrate the case much quicker.